### ISSUE TITLE
[Bug]: When clicking a SVG element within a button element within a Remix-enhanced Form, the name/value on the button is not submitted

### ISSUE THREAD

**Issue by jorupp** [remix-run/remix#1787]
### Which Remix packages are impacted?

- [X] `remix` (Remix core)
- [ ] `create-remix`
- [ ] `@remix-run/architect`
- [ ] `@remix-run/cloudflare-workers`
- [ ] `@remix-run/dev`
- [ ] `@remix-run/express`
- [ ] `@remix-run/netlify`
- [ ] `@remix-run/node`
- [ ] `@remix-run/react`
- [ ] `@remix-run/serve`
- [ ] `@remix-run/server-runtime`
- [ ] `@remix-run/vercel`

### What version of Remix are you using?

dev branch (1.1.3)

### What version of Node are you using? Minimum supported version is 14.

17.4.0

### Steps to Reproduce

Test case: https://github.com/remix-run/remix/pull/1786

Use a SVG element within a button element within a Remix-enhanced Form (normal HTML form works fine)
```
<Form>
  <button name="_action" value="process">
    <svg>....
  </button>
</Form>
```

Click the svg.

### Expected Behavior

When Remix converts this to a `fetch`, the `_action=process` should be included in the post.

### Actual Behavior

When Remix converts this to a `fetch`, the `_action=process` is not included in the post.

**Comment by jorupp** [comment]
I should note that I was able to work around this with a `pointer-events: none` on the SVG element.

**Comment by dougludlow** [comment]
I ran into this too. It was real hard to debug. Thanks for the workaround @jorupp. 👍

**Comment by mcansh** [comment]
this should be resolved as of v1.5.0 (#3094) feel free to reopen if you're still experiencing problems
